app/finders/conversation_finder.rb

```diff
@@ -96,8 +92,15 @@ def filter_by_assignee_type
     @conversations
   end
 
-  def filter_by_reply_status
-    @conversations = @conversations.where(first_reply_created_at: nil) if params[:reply_status] == 'unattended'
+  def filter_by_conversation_type
+    case @params[:conversation_type]
+    when 'mention'
+      conversation_ids = current_account.mentions.where(user: current_user).pluck(:conversation_id)
+      @conversations = @conversations.where(id: conversation_ids)
+    when 'unattended'
+      @conversations = @conversations.where(first_reply_created_at: nil)
+    end
+    @conversations
   end
```

Comment:
@nithindavid As mentioned during our chat, please remove this method and add the logic to find_all_conversations. Also, please add the specs for the same.

Reply:
I have made the changes